.github/workflows/ci.yml

```diff
@@ -7,6 +7,19 @@ on:
       - "api-only"
   pull_request:
     branches: "*"
+    paths-ignore:
+      - "*.md"
+      - LICENCE
+      - TRANSLATION
+      - invidious.service
+      - .git*
+      - .editorconfig
+
```

Comment:
Does Github support the extra space?

Reply:
Yep it does. I'm pretty sure GitHub supports all valid yaml.